packages/core/bootstrap/src/lib/external-adapter/errors.ts

```diff
@@ -24,11 +30,20 @@ export class AdapterError extends Error {
     this.name = name
     this.message = message
     this.cause = cause
+    this.endpoint = endpoint
+    this.errorResponse = errorResponse
+    this.feedID = feedID
   }
 
   toJSONResponse(): AdapterErrorResponse {
     const showDebugInfo = process.env.NODE_ENV === 'development' || process.env.DEBUG === 'true'
-    const errorBasic = { name: this.name, message: this.message }
+    const errorBasic = {
+      name: this.name,
+      message: this.message,
+      endpoint: this.endpoint || undefined,
```

Comment:
It would be better if we can remove `|| undefined` and print exactly what `this.endpoint` is (same for the others).